product_serial/i18n/de.po

```diff
@@ -0,0 +1,221 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+#       * product_serial
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2013-10-09 09:30+0000\n"
+"PO-Revision-Date: 2015-01-31 15:48+0100\n"
+"Last-Translator: Rudolf Schnapka <rs@techno-flex.de>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: de\n"
+"X-Generator: Poedit 1.5.4\n"
+
+#. module: product_serial
+#: field:stock.picking.prodlot.selection,last_number:0
+msgid "Last Number"
+msgstr "Letzte Nummer"
+
+#. module: product_serial
+#: selection:product.product,lot_split_type:0
+msgid "Single"
+msgstr "Einzel"
+
+#. module: product_serial
+#: help:stock.production.lot,last_location_id:0
+msgid "Display the current stock location of this production lot"
+msgstr "Aktuellen Lagerplatz dieses Fertigungsloses anzeigen"
+
+#. module: product_serial
+#: selection:product.product,lot_split_type:0
+msgid "None"
+msgstr "Kein"
+
+#. module: product_serial
+#: model:ir.model,name:product_serial.model_stock_picking_prodlot_selection
+msgid "stock.picking.prodlot.selection"
+msgstr "stock.picking.prodlot.selection"
+
+#. module: product_serial
+#: field:stock.production.lot,last_location_id:0
+msgid "Last location"
+msgstr "Letzter Platz"
+
+#. module: product_serial
+#: view:stock.move:0
+msgid "[('product_id','=?',product_id)]"
```

Comment:
This kind of entry is in .pot file as it is exported by Odoo. So nothing against it as it would appear again next time a translation file is generated.